core/src/main/scala/org/apache/spark/shuffle/FileShuffleBlockManager.scala

```diff
@@ -180,7 +180,8 @@ class FileShuffleBlockManager(conf: SparkConf)
     Some(segment.nioByteBuffer())
   }
 
-  override def getBlockData(blockId: ShuffleBlockId): ManagedBuffer = {
+  override def getBlockData(blockId: ShuffleBlockId,
+      blockManagerId: BlockManagerId = blockManager.blockManagerId): ManagedBuffer = {
```

Comment:
It not support HashShuffle with consolidateShuffleFiles. For not confuse，it only support SortShuffleManager.